nbconvert/utils/exceptions.py

```diff
@@ -13,5 +13,6 @@
 
 class ConversionException(Exception):
     """An exception raised by the conversion process."""
-
-    pass
\ No newline at end of file
+    
+    def __init__(self, message):
+        super(ConversionException, self).__init__(message)
```

Comment:
Is that necessary ?  Wouldn't message be passed to `Exception` if there is no init ?